osf/models/node.py

```diff
@@ -1280,6 +1280,7 @@ def set_privacy(self, permissions, auth=None, log=True, save=True, meeting_creat
                 },
                 auth=auth,
                 save=False,
+                should_hide=True
```

Comment:
Are we sure we always want to hide these logs? Seems like it should only be hidden scenarios where we set privacy ourselves.

Reply:
Good catch. Yeah, let's not hide these logs.